src/mfa/interfaces/challenge-factor-options.ts

```diff
@@ -1,8 +1,4 @@
-export type ChallengeFactorOptions =
-  | {
-      authenticationFactorId: string;
-    }
-  | {
-      authenticationFactorId: string;
-      smsTemplate: string;
-    };
+export interface ChallengeFactorOptions {
```

Comment:
pretty sure this is equivalent but feel free to disagree!

Reply:
I think we should keep it as a union type, as these are mutually exclusive options that could diverge more longer-term.